docs/snippets/vue/page-story.js.mdx

```diff
@@ -0,0 +1,21 @@
+```js
+// Page.stories.js
+
+import Page from './Page.vue';
+import * as HeaderStories from './Header.stories';
+
+export default {
+  title: 'Page',
+};
+
+const Template = (args, { argTypes }) => ({
+  components: { Page },
+  props: Object.keys(argTypes),
+  template: '<page v-bind="$props" />',
+});
```

Comment:
@aach if you don't mind and make some adjustments to the snippet, turning it into:

```js
export default {
  title: 'Page',
  component: Page,
};

const Template = (args, { argTypes }) => ({
  props: Object.keys(argTypes),
  components: { MyPage },
  template: '<my-page :user="user" @onLogin="onLogin" @onLogout="onLogout" @onCreateAccount="onCreateAccount" />',
});
// remainder code doesn't need to change
```

Reasoning behind this is that this snippet is based of the official examples we provide. If we can provide a similar example in the docs as to what the reader gets when he/she installs Storybook it will prevent some misleadings and issues further down the line.

Let me know if you're ok with the changes.

Reply:
@jonniebigodes thank you for your comment.

I tried to replicate the React example as close as possible. 
In that example we are spreading the props so I replicated the same behaviour in the Vue example. I agree that spreading props is more popular in React than in Vue but for this specific example i think it makes sense to do it like that. The point (at least what i understood) of this example is that you are spreading the different args that we have in the Header in the Page (without actually knowing which args and props do we have). 

Also, now that i think about it, you will get all the args from the header and if someone adds one arg to the `Header` that is not present in the `Page` you will see it in the `Page` docs right?

Reply:
@aach i stand corrected. I was thinking about something different, sorry about the confusion. You're correct the way it's now the composed args from `Header` component will propagate into the `Page` and if another one is added into the `Header` component it will show as well.